hpi-schul-cloud/schulcloud-client
# fix announcement styling

# Description

QA note:
I tested the change on env: 
https://brb-bc-4647-fix-link-styling.cd.dbildungscloud.dev/
https://nbc-bc-4647-fix-link-styling.cd.dbildungscloud.dev/
https://default-bc-4647-fix-link-styling.cd.dbildungscloud.dev/
After testing, I have a conflict with the expected result in 2 points, but these 2 points have been accepted by Mona Weitzenberg. 
For more details see jira ticket: https://ticketsystem.dbildungscloud.de/browse/BC-4647 



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-4647
https://github.com/hpi-schul-cloud/dof_app_deploy/pull/583


## Changes


## Data Security


## Deployment


## New Repos, NPM packages or vendor scripts


## Screenshots of UI changes


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
